.github/workflows/monty.yml

```diff
@@ -321,33 +321,27 @@ jobs:
           path: |
             ~/miniconda
           key: ${{ needs.install_monty.outputs.conda_env_cache_key_sha }}
+      - name: Install dependencies
+        # Installing before checkout because GitLFS is missing from the image
+        if: ${{ needs.should_run_monty.outputs.should_run_monty == 'true' }}
+        run: |
+          sudo apt update -y
+          sudo apt install -y --no-install-recommends xvfb libegl1-mesa-dev git-lfs
       - name: Checkout tbp.monty
         if: ${{ needs.should_run_monty.outputs.should_run_monty == 'true' }}
         uses: actions/checkout@v4
         with:
           lfs: true
           path: tbp.monty
-      - name: Install xvfb and mesa
-        # Use X11 framebuffer and mesa opengl libraries for testing.
-        # Not ideal but better than no tests.
-        if: ${{ needs.should_run_monty.outputs.should_run_monty == 'true' }}
-        run: |
-          sudo apt update -y
-          sudo apt install -y --no-install-recommends xvfb libegl1-mesa-dev
       - name: Run python tests
-        # Using 8 CPUs corresponding to the large GithHub-hosted runner available in the "tbp" runner group.
-        # See: https://docs.github.com/en/actions/using-github-hosted-runners/using-larger-runners/about-larger-runners#specifications-for-general-larger-runners
-        #
-        # NOTE: LD_PRELOAD is needed to provide what is discussed in https://stackoverflow.com/questions/71010343/cannot-load-swrast-and-iris-drivers-in-fedora-35/72200748#72200748
         if: ${{ needs.should_run_monty.outputs.should_run_monty == 'true' }}
         working-directory: tbp.monty
         run: |
           export PATH="$HOME/miniconda/bin:$PATH"
           source activate tbp.monty
           set -e
           mkdir -p test_results/pytest
-          export LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libstdc++.so.6
```

Comment:
[I tested this](https://github.com/thousandbrainsproject/tbp.monty/actions/runs/14670677100/job/41177141925) to see if we still need it, and it doesn't seem to be necessary, so I'm removing it.